src/lib/rmt/wizard.rb

```diff
@@ -50,6 +51,7 @@ def run
       'step1' => -> { RMT::WizardSCCPage.new(@config).run },
       'step2' => -> { RMT::WizardMariaDBPage.new(@config).run },
       'step3' => -> { RMT::WizardSSLPage.new(@config).run },
+      'firewall' => -> { RMT::WizardFirewallPage.run(@config) },
```

Comment:
You are passing the @config parameter but it is not defined in the constructor, also commented below

Reply:
Ha, you are watching me ;) Very much apprechiated!
But in this case, I actually had some unpushed changes in the wizard.rb, allowing me to directly start with the firewall page and going to the final page from there, to speed up my try & error :)
I just forgot to push the removed config argument. Sorry, I was not aware that would break it for you. Thanks for pointing it out. Fixed now.